### PR TITLE
OpenAPI to Cloud Armor

### PR DESCRIPTION
**Pull request by cgrotz**
Simple tool that generates Cloud Armor rules from an OpenAPI specification

### CODE REVIEW

File: tools/openapi-to-cloud-armor/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# OpenAPI to Cloud Armor converter
+This tool is a simple converter for OpenAPI specifications to Cloud Armor rules. You can use it as part of your continuous delivery pipeline to configure your Cloud Armor rules based on your OpenAPI spec. This works well if you are developing using an API first approach or if you generate the OpenAPI spec as part of your build process.
+
+To run the tool simply call the following, from within the directory.
+```
+go run main.go -input https://raw.githubusercontent.com/openapitools/openapi-generator/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml
+```
+
+The resulting yaml can be loaded into CloudRun using
```
Comment (boredabdel) [inline tools/openapi-to-cloud-armor/README.md:9]: You mean Cloud Armor ?

**Review by boredabdel — CHANGES_REQUESTED** [review]
One comment, the rest looks good

File: tools/openapi-to-cloud-armor/README.md (hunk 1 above)
Comment (cgrotz) [reply tools/openapi-to-cloud-armor/README.md:9]: yes, oh man 🤣

**Comment by cgrotz** [comment]
@boredabdel thanks, I fixed the typo, sorry about that.

**Review by boredabdel — APPROVED** [review]
LGTM

**Pull request merged**